### ISSUE TITLE
Question about readme

### ISSUE THREAD

**Issue by thewh1teagle** [thedevsbuddy/tailwind_cli#11]
In the readme of this repo you wrote

_"I know there is one package called VelocityX which does the same thing but there are some issues with that package."_
I would like to know what issues `VelocityX` has that `tailwind_cli` solved, and what the advantages and disadvantages of using `tailwind_cli` over `VelocityX`

**Comment by thedevsbuddy** [comment]
@thewh1teagle 

As far as I know, VelocityX is not focusing on the UI only they are providing multiple solutions in 1 package and TailwindCLI is focused only on UI styling. So i will talk about the main similarity about TailwindCLI and VelocityX, and that is providing Tailwind like styling for flutter and when I checked VelocityX last time they didn't have any option for customisation of any styling and also we don't have fully control on some properties like margins, paddings etc.

So I decided to create this project just to be focused only on the UI styling.

Hope this will be helpful to you.
